### PR TITLE
Update firefox_cloud_services_ToS/cs.md

### PR DESCRIPTION
**Pull request by MikkCZ**
- Účty Firefox -> Účet Firefoxu (correct brand l10n, multiple times in the text)
- budete muset založit -> si budete muset založit (missing pronoun)
- pro činnosti prováděné -> nesete odpovědnost za činnosti (partial revert of https://github.com/mozilla/legal-docs/commit/b8d7aca10aa67c6f044270c8008ae3a2fee5cfde, the sentence does not make sense)
- pomocí části Možnosti -> v Možnostech (more natural wording)
- Najdi mé zařízení -> Najít mé zařízení (correct feature name)
- v nastaveních zařízení -> v nastavení zařízení (in Czech we prefer singular instead of plural in this case)
- Aplikace Firefox Screenshots -> Služby Firefox Screenshots (it's a service, not an app)
- Viz zde kde můžete -> Zde můžete (more natural wording)
- jak s těmito informacemi zacházíme -> jak tyto informace používáme (better wording more consistent with other documents)
- Zásady ochrany osobních údajů (the word Mozilla is not needed, as the appropriation to Mozilla is clear by naše (=our) )
- Obsah můžete nahrávat v rámci využití funkcí Služeb. -> V rámci využívání funkcí Služeb můžete nahrávat svůj obsah. (more natural wording)
- obchodních firem (that would mean a "trade company")
- název domén -> názvů domén (declension)
- Veřejné licence Mozilla -> Mozilla Public Licence (that's a name of the license, do not translate it)
- maximální úsilí (that would mean maximal effort)

+ locale non-specific links

### CODE REVIEW

**Comment by peiying2** [comment]
@MikkCZ let me know if I can merge this PR or if this is subject for peer review before I merge it.

**Comment by MikkCZ** [comment]
@peiying2 Good point, I should ask someone for a review if i haven't missed anything. Especially for the big pull requests. If there is any production push waiting for this, I think you can go ahead and postpone these changes for the next push. The current texts are there for some time already.

**Comment by peiying2** [comment]
Thanks for the fixes on the URLs. I will merge those but leave this for peer review. 

Currently the legal pushes to production is manual. Kohei has been helping me with on the dev side.  Let me know when this is ready.  We will include it whenever the next push is.

File: firefox_cloud_services_ToS/cs.md [hunk 1]
```diff
@@ -13,39 +13,39 @@ Tato úvodní část je shrnutím níže uvedených ustanovení. Toto shrnutí s
 
     Tyto podmínky poskytování služeb (dále jen „<u>Podmínky</u>”) upravují vaše používání Firefox cloud services, souboru online služeb poskytovaných společností Mozilla (dále jen „Služby”).
 
-2. #### Účty Firefox
+2. #### Účet Firefoxu
 
-    Pro používání některých Služeb budete muset založit účet Firefox.  Během registrace budete požádáni o nastavení hesla. Jste povinni uchovat své heslo v důvěrnosti a pro činnosti prováděné prostřednictvím vašeho účtu Firefox. Společnost Mozilla nenese odpovědnost za žádné ztráty, k nimž dojde během neoprávněného používání vašeho účtu Firefox.
+    Pro používání některých Služeb si budete muset založit účet Firefox. Během registrace budete požádáni o nastavení hesla. Jste povinni uchovat své heslo v důvěrnosti a nesete odpovědnost za činnosti prováděné prostřednictvím vašeho účtu Firefoxu. Společnost Mozilla nenese odpovědnost za žádné ztráty, k nimž dojde během neoprávněného používání vašeho účtu Firefoxu.
```
Comment (FHRNet) [inline firefox_cloud_services_ToS/cs.md]: "účet Firefox" => "účet Firefoxu".
Otherwise looks good

Comment (peiying2) [reply firefox_cloud_services_ToS/cs.md]: Firefox is a brand name, it needs to remain in English, per branding guideline. I will let @MikkCZ comment on your suggestions.

Comment (MikkCZ) [reply firefox_cloud_services_ToS/cs.md]: Hi @peiying2. "Firefox" is a brandname, but from what I know, "Firefox Accounts" are allowed to be localized. In [bug 1317086](https://bugzilla.mozilla.org/show_bug.cgi?id=1317086) we have agreed on translating "Firefox Accounts" as "účet Firefoxu" (the Firefox name is declined there) and all the material already refer to it the new name for some time. In Czech it's impossible to have a natural translation for Firefox Accounts without declension.

Comment (peiying2) [reply firefox_cloud_services_ToS/cs.md]: ah, sorry for injecting comments not knowing the background.  Sounds good.

Comment (MikkCZ) [reply firefox_cloud_services_ToS/cs.md]: No problem. :) I see this as an issue and I often need to check the documentation to remind, what we should translate and what we should not...

I will now go ahead an address the issues pointed above.

**Pull request merged**